node/src/components/chain_synchronizer/error.rs

```diff
@@ -32,13 +32,16 @@ pub(crate) enum Error {
     ),
 
     #[error(
-        "cannot get trusted validators for such an early era. \
+        "trusted header is from before the last upgrade and isn't the last header before \
+         activation. \
          trusted header: {trusted_header:?}, \
-         last emergency restart era id: {maybe_last_emergency_restart_era_id:?}"
+         protocol_version: {protocol_version:?}, \
+         activation_point: {activation_point:?}"
```

Comment:
It's not obvious to me, just from this error message, what's the current networks protocol version and what is trusted header's.

Reply:
Since the trusted header will be printed whole, the header's protocol version will be a field in the header.
But if you think it's better to eg. rename `protocol_version` to `current_protocol_version`, I could do that.

Reply:
So maybe something like:
```
trusted header is from before the last upgrade ({activation_point:?}). current protocol version is: {protocol_version:?}
```
?
This way node operators knows when the upgrade happened (what era), can compare what was the era of the trusted header provided and what's the current version.

Reply:
The thing is, I wanted to avoid phrasing the error as "trusted header is from before the last upgrade", because that's still allowed if we're currently upgrading. Then the trusted header is the last header from before the upgrade, and we're going to commit the upgrade and construct the immediate switch block.

Reply:
I see... so let's go with what you wanted - rename to `current protocol version`. We can drop the underscores in the error message.